dotcom-rendering/src/web/layouts/StandardLayout.tsx

```diff
@@ -411,42 +413,37 @@ export const StandardLayout = ({ CAPIArticle, NAV, format }: Props) => {
 							editionId={CAPIArticle.editionId}
 						/>
 					</Section>
-					{NAV.subNavSections &&
-						format.theme !== ArticleSpecial.Labs && (
-							<>
-								<Section
-									fullWidth={true}
-									backgroundColour={
-										palette.background.article
-									}
-									padSides={false}
-									element="aside"
-								>
-									<Island deferUntil="idle">
-										<SubNav
-											subNavSections={NAV.subNavSections}
-											currentNavLink={NAV.currentNavLink}
-											format={format}
-										/>
-									</Island>
-								</Section>
-								<Section
-									fullWidth={true}
-									backgroundColour={
-										palette.background.article
-									}
-									padSides={false}
-									showTopBorder={false}
-								>
-									<StraightLines
-										count={4}
-										cssOverrides={css`
-											display: block;
-										`}
+					{NAV.subNavSections && !isLabs && (
```

Comment:
Is this intentional? The PR description only mentions removing the merchandising ad slots for Labs articles but here we are also removing other content

Reply:
Ah, wait, I see. This isn't changing behaviour, it's because you've created the `isLabs` variable. Carry on.